WegliKit/Sources/CameraAccessClient/Mock.swift

```diff
@@ -0,0 +1,9 @@
+import ComposableArchitecture
+import Photos
+
+public extension CameraAccessClient {
+  static let mock = Self(
+    requestAuthorization: { .none },
+    authorizationStatus: { .notDetermined }
+  )
+}
```

Comment:
```suggestion
public extension CameraAccessClient {
  static let mock = Self(
    requestAuthorization: { .none },
    authorizationStatus: { .notDetermined }
  )
  
  static let authorized = Self(
    requestAuthorization: { return Just(true).eraseToEffect() },
    authorizationStatus: { .authorized }
  )
  
  static let denied = Self(
    requestAuthorization: { return Just(false).eraseToEffect() },
    authorizationStatus: { .denied }
  )
}
```

could add some convenience ones for testing